libs/aws/tests/integration_tests/vectorstores/test_documentdb.py

```diff
@@ -0,0 +1,728 @@
+"""Test DocumentDBVectorSearch functionality."""
+import logging
+import os
+from asyncio import sleep as asyncio_sleep
+from time import sleep
+from typing import Any, Optional, Tuple
+
+import pytest
+from langchain_core.documents import Document
+from langchain_openai.embeddings import OpenAIEmbeddings
+from motor.core import AgnosticCollection
+from pymongo.collection import Collection
+
+from langchain_aws.vectorstores.documentdb import (
+    DocumentDBSimilarityType,
+    DocumentDBVectorSearch,
+)
+
+logging.basicConfig(level=logging.DEBUG)
+
+model_deployment = os.getenv(
+    "OPENAI_EMBEDDINGS_DEPLOYMENT", "smart-agent-embedding-ada"
+)
+model_name = os.getenv("OPENAI_EMBEDDINGS_MODEL_NAME", "text-embedding-ada-002")
+
+INDEX_NAME = "langchain-test-index"
+NAMESPACE = "langchain_test_db.langchain_test_collection"
+CONNECTION_STRING = os.getenv("DOCUMENTDB_URI", "")
+DB_NAME, COLLECTION_NAME = NAMESPACE.split(".")
+
+dimensions = 1536
+similarity_algorithm = DocumentDBSimilarityType.COS
+
+
+def prepare_collection() -> Tuple[Collection, AgnosticCollection]:
+    from motor.motor_asyncio import AsyncIOMotorClient
+    from pymongo import MongoClient
+
+    test_client: MongoClient = MongoClient(CONNECTION_STRING)
+    test_async_client: AsyncIOMotorClient = AsyncIOMotorClient(CONNECTION_STRING)
+    return test_client[DB_NAME][COLLECTION_NAME], test_async_client[DB_NAME][
+        COLLECTION_NAME
+    ]
+
+
+@pytest.fixture()
+def collections() -> Any:
+    return prepare_collection()
+
+
+@pytest.fixture()
+def embedding_openai() -> Any:
+    openai_embeddings: OpenAIEmbeddings = OpenAIEmbeddings(
+        deployment=model_deployment, model=model_name, chunk_size=1
+    )
+    return openai_embeddings
+
+
+"""
+This is how to run the integration tests:
+
+cd libs/community
+make test TEST_FILE=tests/integration_tests/vectorstores/test_documentdb.py
```

Comment:
Needs an update to refer to `langchain-aws`. Were you able to run these tests and confirm these run successfully after copying the files here?

Reply:
Changed